web/libs/core/src/lib/utils/dom.ts

```diff
@@ -96,6 +96,7 @@ export const alignElements = (
   align: Align = "bottom-left",
   padding = 0,
   constrainHeight = false,
+  openUpwardForShortViewport = true,
```

Comment:
It feels it should not be an option or a variable but the normal behaviour meaning that, no matter the position the client selects, if there is overflow because of the viewport in any of the 4 directions, it should be added on a different direction by default

Reply:
went ahead with merge since this fix is high priority